src/workerd/api/tests/js-rpc-test.js

```diff
@@ -165,6 +204,28 @@ export let namedServiceBinding = {
     // Properties that return a function can actually be called.
     assert.strictEqual(await env.MyService.functionProperty(19, 6), 13);
 
+    // Members of an object-typed property can be invoked.
+    assert.strictEqual(await env.MyService.objectProperty.func(6, 4), 24);
+    assert.strictEqual(await env.MyService.objectProperty.deeper.deepFunc(6, 3), 2);
+    assert.strictEqual(await env.MyService.objectProperty.counter5.increment(3), 8);
```

Comment:
non-blocking nit: for completeness, let's have a test that uses `Promise.all(...)`, `Promise.any`, `Promise.race`, and `Promise.allSettled` with these to ensure the concurrent outstanding requests have no problem.

Reply:
I've added a test of concurrent calls which uses `Promise.all()`. I don't see a need to test all the different `Promise` helpers since there's no reason to expect any of them behave unusually on thenables.